nbs/12_cli.ipynb

```diff
@@ -61,6 +61,14 @@
     "> CLI commands"
```

Comment:
<p>Some of these new docs look a bit too long to me. Maybe move some details into markdown?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/fastai/nbdev/pull/866/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='52'/>

Reply:
I was trying something out there :D I think it looks decent in docs, but definitely looks worse in the code! Will move to markdown

<img width="768" alt="image" src="https://user-images.githubusercontent.com/559360/184853970-6f81ea5e-fdac-4702-bad2-e64c406d0519.png">